exercicioLogica5/src/main/java/org/example/TabuadaServiceTest.java

```diff
@@ -0,0 +1,23 @@
+package org.example;
+
+import org.junit.Test;
+import org.junit.jupiter.api.Assertions;
+
+
+
+public class TabuadaServiceTest {
+
+    TabuadaService tabuadaService = new TabuadaService();
+    int numero = 5;
+    int multiplicador;
+    int resultadoFinal;
+    @Test
```

Comment:
Teste está ok, agora entendi o por quê do resultadoFinal, seria para ver se o teste funcionou e chegou no mesmo resultado. Porém, numa tabuada, digamos que tem vários resultadosFinais, 1x5 o resultado final é 5, 2x5 o resultado é 10 e assim em diante. Então, uma sugestão de como você poderia fazer, talvez você deveria colocar uma lista e comparar todos os resultados para ver se ficou correto. Ou então ao invés de dar um print dentro do teu for, você poderia guardar dentro de uma string e depois fazer um assertEquals digitando uma tabuada e comparando as strings.